root/usr/share/nethesis/NethServer/Help/en/NethServer_Module_TlsPolicy.rst

```diff
@@ -0,0 +1,7 @@
+========================
+Transport Layer Security
+========================
+
+TLS is a cryptographic protocols that provide communications security over a computer network. Several versions of the protocols find widespread use in applications such as web browsing, email, Internet faxing, instant messaging, and voice over IP (VoIP). The TLS protocol aims primarily to provide privacy and data integrity between two communicating computer applications.
+
+You can enforce the TLS policy, or choose the legacy one if your clients are not supported/maintained anymore (Windows XP for example)
```

Comment:
We could apply to new modules a new inline help pattern... I'm pushing this approach

https://github.com/NethServer/nethserver-samba/blob/master/root/usr/share/nethesis/NethServer/Help/en/NethServer_Module_FileServer.rst

A link to a special admin manual section. The main reason is: the manual is translated on Transifex

Reply:
done, see my PR in the doc side